### PR TITLE
add retirement landing page

### PR DESCRIPTION
**Pull request by asmega**

### CODE REVIEW

File: spec/controllers/retirements_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+RSpec.describe RetirementsController, type: :controller, features: [:rio] do
+  describe '#index' do
+    it 'works' do
```
Comment (esbel) [inline spec/controllers/retirements_controller.rb]: change it something than includes success or 200 :D

**Comment by esbel** [comment]
:+1:

**Comment by Tr4pSt3R** [comment]
Might want to add this to the controller test? 
```
expect(response).to render_template("index")
```
just an addition. All looks good to go. :+1:

**Comment by asmega** [comment]
If you delete the view right now the test will probably fail, so it would be a redundant test

**Comment by asmega** [comment]
Thanks!

**Comment by Tr4pSt3R** [comment]
:+1: :100:

**Pull request merged**